### DISCUSSION TITLE
The S-132 add-on is available to order.

### DISCUSSION THREAD

**Discussion by dmcnaugh** [thehighnibble/thehighnibble.github.io#34] [Announcements]
Pricing is published on the website at https://thehighnibble.com/s132/#availability
so I won't repeat it here.

Kits will start shipping this month.

**Comment by GrantMeStrength** [comment]
Hi - any chance of a nice detailed photo of the finished board, so I can confirm that I have the components in the right way around? (Especially the SMT ones - the logic chips don’t have dimples as far as I can see, so I’m uncertain as to their orientation). Thanks!

**Reply by dmcnaugh** [reply]
I've added some build notes to the S-132 site along with the detailed photo as requested.
See: https://thehighnibble.com/s132/build/

**Reply by GrantMeStrength** [reply]
Thank you - that was a great help, and It all went together quite painlessly. A "fun build", as they say. Moving to soldering the regular ICs after working on the SMTs is such a funny experience

**Comment by nealzip** [comment]
On the S-132 Board. Has anyone used a VGA to HDMI cable to connect to my HDMI Computer monitor? Or do I need to have a True VGA Monitor to use the S-132?
